pluginClientSdkLib/build.gradle

```diff
@@ -17,6 +17,8 @@ dependencies {
     compile group:'com.twofortyfouram', name:'android-spackle', version:'[1.0.4,2.0['
 
     androidTestCompile group:'com.twofortyfouram', name:'android-test', version:'[1.0.4,2.0['
+    testCompile fileTree(dir: "$rootProject.projectDir/build/intermediates/exploded-aar/com.android.support/support-v4/", include: "**/*.jar")
```

Comment:
This question is still open: what is the testCompile option here supposed to do?